gaurun/conf.go

```diff
@@ -42,6 +42,9 @@ type SectionIos struct {
 	PemCertPath      string `toml:"pem_cert_path"`
 	PemKeyPath       string `toml:"pem_key_path"`
 	PemKeyPassphrase string `toml:"pem_key_passphrase"`
+	TokenAuthKeyPath string `toml:"token_auth_key_path"`
+	TokenAuthKeyID   string `toml:"token_auth_key_id"`
+	TokenAuthTeamID  string `toml:"token_auth_team_id"`
```

Comment:
Separating struct looks to be better. (Though the same applies Pem(Cert|Key)Path and PemKeyPassphrase.)

```
type TokenAuth struct {
    KeyPath string
    ...
}
```

Reply:
Does this PR include the change of `Pem(Cert|Key)Path`?

Reply:
Either is fine.

Reply:
I would like to make a minimum change for this PR. So, I will change only about `TokenAuth`.

Reply:
I think the present expression is good for `toml` expression.